android/app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java

```diff
@@ -67,9 +68,10 @@ public void onRefresh() {
         sponsorsRecyclerView.addOnItemTouchListener(new RecyclerItemClickListener(view.getContext(), new RecyclerItemClickListener.OnItemClickListener() {
             @Override
             public void onItemClick(View itemView, int position) {
-                String sponsorUrl = dbSingleton.getSponsorList().get(position).getUrl();
-                if (!sponsorUrl.startsWith("http") || !sponsorUrl.startsWith("https")) {
+                String sponsorUrl = dbSingleton.getSponsorList().get(position).getLink();
+                if (!sponsorUrl.startsWith("http")) {
                     sponsorUrl = "http://" + sponsorUrl;
```

Comment:
Yep i get it, but it was doing this.
Hence as a result all of the links were invalid unless i removed this.